app/Http/Controllers/BuildMetricController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\BuildMetric;
+use Artesaos\SEOTools\Facades\SEOTools;
+use Illuminate\Http\Request;
+
+class BuildMetricController extends Controller
+{
+    public function index(Request $request, $gun, $combo)
+    {
+        // SEOTools::setTitle('Deep Rock Galactic Loadouts');
+        // TODO: SEO
+
+        $build = BuildMetric::where([
+            ['gun_id', '=', $gun],
+            ['build_combination', '=', $combo],
+        ])->firstOrFail();
+
+        $build_gun = $build->gun;
+
+        $mod_matrix = $build->getModMatrix($build_gun, $combo);
+        $combo_array = str_split($combo);
+        $overclock = false;
+        if ($combo_array[5] != '-') {
+            $overclock = $build->getOverclock($gun, $combo_array[5]);
```

Comment:
Could pass the `$build_gun` to `$build->getOverclock` so we don't have to query on the other side. Would be a bit faster.

Reply:
Can you please clarify

Reply:
Not sure I can clarify any further, but I can jump on a call and explain tomorrow if you want.